packages/airnode-protocol/hardhat.config.js

```diff
@@ -6,7 +6,7 @@ require('hardhat-gas-reporter');
 const api3Chains = require('@api3/chains');
 require('dotenv').config();
 
-const { apiKey: etherscanApiKey, customChains: etherscanCustomChains } = api3Chains.hardhatEtherscan();
+const { apiKey: etherscanApiKey, customChains: etherscanCustomChains } = api3Chains.hardhatConfig.etherscan();
```

Comment:
It should be possible to remove all of this boilerplate code and just have:

```ts
const etherscan = api3Chains.hardhatConfig.etherscan();
const networks = api3Chains.hardhatConfig.networks();

module.exports = {
  ...
  etherscan,
  networks,
};
```

I haven't tested this though. Maybe @bbenligiray can confirm

Reply:
Yes, see https://github.com/api3dao/merkle-funder/blob/main/hardhat.config.ts

Reply:
Thanks- I removed the boilerplate